### PR TITLE
Update

### LINKED ISSUE #195: Can onlly use chgcentre docker image on machine it was built on
see https://sourceforge.net/p/wsclean/tickets/143/ for more details

### PR DESCRIPTION
**Pull request by Athanaseus**
Build binary that is portable on multiple platforms.

resolves https://github.com/kernsuite/packaging/issues/195

### CODE REVIEW

**Comment by gijzelaerr** [comment]
where is the debian branch PR?

**Comment by Athanaseus** [comment]
this is the debian branch PR, it's not really a new release but a package update. `1kern2`

File: debian/rules [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/make -f
 
+CMAKE_FLAGS += -DDEB_BUILD_MULTIARCH=`dpkg-architecture -qDEB_BUILD_MULTIARCH` -DPORTABLE=yes
```
Comment (gijzelaerr) [inline debian/rules:3]: what does the `dpkg-architecture -qDEB_BUILD_MULTIARCH` do?

Comment (Athanaseus) [reply debian/rules:3]: It's supposed to determine the architecture of the current host system where the debian package is created.

For instance, in my case, this results in:
```
$ dpkg-architecture --query DEB_HOST_MULTIARCH
x86_64-linux-gnu
```
So this should support the cross-compilation of this package.

**Comment by gijzelaerr** [comment]
So that is quite smart, but a bit too smart :) now we depend on the build architecture, which could be never than the runtime architecture. We want to be building for quite a conservative target.

That said, I think the ` -DPORTABLE=yes` flag might be enough already?

**Comment by Athanaseus** [comment]
Ok you are right that is not really necessary since we want to build for a conservative target.

> That said, I think the -DPORTABLE=yes flag might be enough already?

Yes this should make the binary usable on different platforms. I guess that was the main reason for this update. I'll remove that.

PS: I initially saw this technique being used in the `wsclean/debian/rules` file.

**Comment by gijzelaerr** [comment]
ah! maybe i'm actually wrong then and it is useful

**Pull request merged**